pages/phases/assessments/training.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: article
-permalink: /hiring-phases/phone-assessment-interviews/training/
+permalink: /hiring-phases/assessments/training/
 section: hiring-phases
 sidenav: hiring-phases
-phase: phone-assessment-interviews
+phase: assessments
 sub-phase-order: 015
-title: Training SMEs to Conduct Interviews
-intro: While phone interviews are being scheduled with applicants, facilitate a mandatory two-hour interview training for all SMEs who will conduct the next round of interviews. The training should explain how to conduct structured interviews, and then assess the applicant’s proficiency with the required competencies based on their interview responses.
+title: Training SMEs to Conduct Assessments
+intro: Facilitate a mandatory two-hour training for all SMEs participating in an assessment round. The training should explain how to review the assessment (including conducting an interview, and how to assess the applicant’s proficiency levels.
```

Comment:
how to review the assessment against the required proficiency levels (structure interview, written prompt, or demonstration project)

Reply:
updated here (I took out the assessment options because those should be named elsewhere): https://github.com/usds/SME-QA-Website/pull/243/commits/f5c3cc1cf0455a1a911bec492f97f0c513337dd6

We will need to add assessment training content here, I made https://github.com/usds/SME-QA-Website/issues/246 to track that.

Reply:
Also, good catch on me never closing the parenthesis there.